src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_ISwitchOperation.cs

```diff
@@ -1972,11 +1972,11 @@ void M(bool result, dynamic input)
                 IParameterReferenceOperation: input (OperationKind.ParameterReference, Type: dynamic) (Syntax: 'input')
 
         Jump if False (Regular) to Block[B3]
-            IIsPatternOperation (OperationKind.IsPattern, Type: System.Boolean, IsImplicit) (Syntax: 'case 1:')
+            IIsPatternOperation (OperationKind.IsPattern, Type: System.Boolean, IsImplicit) (Syntax: '1')
               Expression: 
                 IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: dynamic, IsImplicit) (Syntax: 'input')
               Pattern: 
-                IConstantPatternOperation (OperationKind.ConstantPattern, Type: null, IsImplicit) (Syntax: 'case 1:')
+                IConstantPatternOperation (OperationKind.ConstantPattern, Type: System.Int32, Constant: 1, IsImplicit) (Syntax: '1')
```

Comment:
>System.Int32, Constant: 1 [](start = 80, length = 25)

This change is unexpected, ```IConstantPatternOperation``` doesn't represent a value, therefore it should never have type or a constant value.